documentation/modules/auxiliary/scanner/sap/sap_redwood_traversal.md

```diff
@@ -0,0 +1,36 @@
+# Description
+This is a module for exploitation an anon directory traversal vulnerability in SAP NetWeaver Application Server Java in Redwood module. The vulnerability was active exploited from Chines hackers in the wild in 2017.
+
+
+# Verification Steps
+```
+Start msfconsole
```

Comment:
This should be a numbered list.

Also, real examples are nice, ie `set FILEPATH /etc/passwd`.

```
1. Start msfconsole
2. use auxiliary/scanner/sap/sap_redwood_traversal
3. set RHOSTS [IP]
4. set FILEPATH /etc/passwd
5. set DEPTH 15
6. run
```